hapi/contact_lists.py

```diff
@@ -15,13 +15,28 @@ def __init__(self, *args, **kwargs):
         super(ContactListsClient, self).__init__(*args, **kwargs)
         self.log = logging_helper.get_log('hapi.contact_lists')
 
-    def _get_path(self, subpath):
-        return 'contacts/v%s/%s' % (self.options.get('version') or CONTACT_LISTS_API_VERSION, subpath)
+    def _get_path(self, method):
```

Comment:
Hmm I saw in broadcast.py that they used method as this name, but I only looked there. I'll change back to subpath.